rust/pact_verifier_cli/src/args.rs

```diff
@@ -1,28 +1,38 @@
-use clap::{App, Arg};
+use clap::{App, Arg, ArgAction};
 use regex::Regex;
 
-fn port_value(v: String) -> Result<(), String> {
+fn port_value(v: &str) -> Result<(), String> {
   v.parse::<u16>().map(|_| ()).map_err(|e| format!("'{}' is not a valid port value: {}", v, e) )
 }
 
-fn integer_value(v: String) -> Result<(), String> {
+fn integer_value(v: &str) -> Result<(), String> {
   v.parse::<u64>().map(|_| ()).map_err(|e| format!("'{}' is not a valid integer value: {}", v, e) )
 }
 
-pub(crate) fn setup_app<'a, 'b>(program: String, version: &'b str) -> App<'a, 'b> {
+fn transport_value(v: &str) -> Result<(String, u16), String> {
```

Comment:
Is it possible for a transport to be a socket? (we don't need to tackle it now if it can be added).

Reply:
Sockets still need to use a port. But the verifier won't know how to write the request or read the response, so it will have to be done with a plugin